### ISSUE TITLE
Ksw toolkit 2 and Snapdragon 665

### ISSUE THREAD

**Issue by cpolhout** [Snaggly/KSW-ToolKit#22]
Hi, I try to use ksw-toolkit2 app from the playstore. But it seems Snapdragon 665 - unit is not supported. Can you confirm that? Do you expect it to be supported? Or are you looking for beta testers with that unit?

(It's this one: https://www.android-car-dvd.de/collections/android-head-unit-1/products/10-25-android-10-0-4g-64g-qualcomm-8-core-built-in-4g-lte-ips-car-multimedia-for-bmw-x5-e70-x6-e71-2007-2014?variant=40201296674949 )

**Comment by Snaggly** [comment]
Correct, only Snapdragon 625, 662 and 680 are supported. 450 might be too but it's not fully tested. I had looked into that but it seems very to stay unsupported for the coming future.

**Comment by cpolhout** [comment]
Thanks for your answer. Is there an option I can test it on my unit? So test a version were the check is removed? (Or share the source code, so I could fork it). Thanks in advance!
